src/test/java/ActionableSmartHomeTest.java

```diff
@@ -0,0 +1,127 @@
+import org.junit.Assert;
+import org.junit.Test;
+import ru.sbt.mipt.oop.Action;
+import ru.sbt.mipt.oop.HomeProvider.HomeProvider;
+import ru.sbt.mipt.oop.HomeProvider.JsonHomeProvider;
+import ru.sbt.mipt.oop.SmartHome.Door;
+import ru.sbt.mipt.oop.SmartHome.Light;
+import ru.sbt.mipt.oop.SmartHome.Room;
+import ru.sbt.mipt.oop.SmartHome.SmartHome;
+import ru.sbt.mipt.oop.SmartHomeUtility;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+
+public class ActionableSmartHomeTest {
+
+    HomeProvider homeProvider;
+    SmartHome smartHome;
+
+    public void setHome() {
+        homeProvider = new JsonHomeProvider("smart-home-1.js");
+        smartHome = homeProvider.provideHome();
+    }
+
+    @Test
+    public void ActionableLightsTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.